### PR TITLE
547295 Provide javadoc for org.eclipse.passage.lic.api.restrictions package

### PR DESCRIPTION
**Pull request by eparovyshnaya**
Signed-off-by: Elena Parovyshnaia <elena.parovyshnaya@gmail.com>

### CODE REVIEW

**Comment by eparovyshnaya** [comment]
Aimed to close [547295](https://bugs.eclipse.org/bugs/show_bug.cgi?id=547295)

File: bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java [hunk 1]
```diff
@@ -15,17 +15,32 @@
...
+ * The service implementing the <i>restrictions execution</i> phase
+ * of <i> access cycle</i>.
+
  * @since 0.4.0
+ * @see org.eclipse.passage.lic.api
  */
 public interface RestrictionExecutor {
 
+	/**
+	 * <p>Applies the given {@link RestrictionVerdict}s for feature under licencing
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java]: licencing -> licensing

File: bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java [hunk 2]
```diff
@@ -15,17 +15,32 @@
...
+	 *
+	 * <p>Is capable to affect the program runtime environment the following ways:</p>
+	 * <ol>
+	 *    <li>early exit from command line tools with notice</li>
+	 *    <li>block dialogs for UI application</li>
+	 *    <li>filter out the UI</li>
+	 *    <li>block of bundles using OSGi level</li>
+	 * </ol>
+	 *
+	 * @param verdicts restriction instructions gained at the pagse of <i>permission examining</i> phase of <i>access cycle</i>
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java]: pagse -> phase ?

File: bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java]: Done

File: bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/restrictions/RestrictionExecutor.java]: Rewrited

**Pull request merged**